src/gtfs/repository/CIFRepository.ts

```diff
@@ -117,13 +115,11 @@ WHERE
 	(sloc.schedule_location_id IS NULL OR (loc.crs_code IS NOT NULL AND loc.crs_code != "") )
 	AND s.wef_date < ?
   AND s.weu_date >= ?
-  AND (s.import_weu_date IS NULL OR (s.import_weu_date > ?) )
+  AND (s.import_weu_date IS NULL OR (s.import_weu_date > ?) ) AND (s.schedule_type != 'VSTP' OR ?)
   
   HAVING runs_to >= runs_from
 ORDER BY stp_indicator DESC, s.schedule_id, sloc.location_order
-
-
-      `, [this.endRange, this.startRange, this.startRange]);
+      `, [this.endRange.format("YYYY-MM-DD"), this.startRange.format("YYYY-MM-DD"), this.startRange.format("YYYY-MM-DD"), !this.excludeVstpSchedules]);
```

Comment:
The `startRange` and `endRange` became `moment` after the recent date validation update, and it start to throw exception when deploy and test on my local machine:
```
UnhandledPromiseRejectionWarning: Error: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ' `_i` = '2019-04-25', `_f` = 'YYYY-MM-DD', `_isUTC` = false, `_pf` = '[object Ob' at line 25
```
It seems like that at this line (and line 172), the raw `moment` object has been passed into the sql query template and causing the sql syntax error, therefore I had to switch it back to string here to fix the issue.

Reply:
Ok, I'm fine with that.